platform/src/test/kotlin/org/stellar/anchor/platform/LogAppenderTest.kt

```diff
@@ -10,10 +10,12 @@ import org.apache.logging.log4j.core.config.LoggerConfig
 import org.junit.jupiter.api.AfterEach
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.params.ParameterizedTest
 import org.junit.jupiter.params.provider.CsvSource
 import org.stellar.anchor.util.*
 
+@Disabled
```

Comment:
Should this be disabled?

Reply:
Yes, this is interfering with all other tests with Logger calls. When we mock them, some other tests may fail. I temporarily disabled this and added a ticket to re-enable them. Probably need to use TestSuite so that they won't run with other tests.

Reply:
ANCHOR-509